website/static/js/registrationUtils.js

```diff
@@ -439,6 +439,8 @@ var RegistrationEditor = function(urls, editorId) {
     });
     self.showValidation = ko.observable(false);
 
+    self.contributors = ko.observable(self.getContributors());
```

Comment:
We typically use knockout observables when a value is expected to change (in ways that will affect other things- the change is "observed")

If this is a simple static value, it can just be a regular attribute; no need to use `ko.observable`